scaleway/resource_instance_security_group.go

```diff
@@ -368,7 +368,7 @@ func securityGroupRuleSchema() *schema.Resource {
 			},
 			"port_range": {
 				Type:        schema.TypeString,
-				Computed:    true,
+				Optional:    true,
```

Comment:
Why not `Optional` and `Computed`?

Reply:
It is never computed only the one you use will  be store in state either `port` or `port_range` but never both